server/subsonic/media_retrieval.go

```diff
@@ -95,9 +95,9 @@ func (api *Router) GetLyrics(r *http.Request) (*responses.Subsonic, error) {
 	artist, _ := p.String("artist")
 	title, _ := p.String("title")
 	response := newResponse()
-	lyrics := responses.Lyrics{}
-	response.Lyrics = &lyrics
-	mediaFiles, err := api.ds.MediaFile(r.Context()).GetAll(filter.SongWithLyrics(artist, title))
+	lyricsResponse := responses.Lyrics{}
+	response.Lyrics = &lyricsResponse
+	mediaFiles, err := api.ds.MediaFile(r.Context()).GetAll()
```

Comment:
Any particular reason why you removed the filter?

Reply:
Yes, the filter only works when there are embedded lyrics and would preclude searching for lyrics by file (or, plugin) when that comes.

Reply:
Ah, I do see the need for artist/title filter. Let me add that test.